determined-ai/determined
# feat: task specific actions to job overflow menu

## Description
[DET-7077]



## Test Plan

- Check if there are `view log` and `kill` buttons in the `notebooks`, `tensor boards`, `commands`, and `shells` job types and check the functionalities
- Check if there are `Cancel` and `kill` buttons in the `experiment` job type and check the functionalities




## Commentary (optional)




## Checklist

See [Release Note](../docs/release-notes/README.md) for details.





[DET-7077]: https://determinedai.atlassian.net/browse/DET-7077?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
